bundles/org.openhab.binding.serial/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -26,8 +26,10 @@
 				<advanced>true</advanced>
 				<label>Baud Rate</label>
 				<description>Set the baud rate</description>
+				<limitToOptions>false</limitToOptions>
```

Comment:
When I tried adding this it stops the UI displaying the options and just presents a box to type in any number. What do you see?

Reply:
Yes, apparently in that case, Main UI does not show the options. But at least, you can enter any baudrate.